src/instruments/src/EFB/ATC/ATC.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import * as apiClient from '@flybywiresim/api-client';
+import { IconBuildingLighthouse, IconChartRadar, IconCircleCheck, IconPlaneArrival, IconPlaneDeparture, IconTrafficLights } from '@tabler/icons';
+import { useSimVar, useSplitSimVar } from '../../Common/simVars';
+import './ATC.scss';
```

Comment:
Could've been done without the css file?

Tailwind classes can be directly applied to divs. In case, you're using the CSS file, please use `@apply` directive.